app/views/landing-pages/parents/PageParents.vue

```diff
@@ -706,9 +709,13 @@ export default {
       $("#student-outcome-carousel").carousel(frameNum)
     },
 
-    onClickMainCta () {
+    async onClickMainCta () {
       this.trackCtaClicked()
-      if (this.scheduleFreeClassExperiment === 'schedule-free-class') {
+
+      const { isAvailable, availabilityPDT } = await getAvailability()
+      this.availabilityPDT = availabilityPDT
+
+      if (isAvailable && await this.scheduleFreeClassExperiment === 'schedule-free-class') {
```

Comment:
do we want to continue with ABTest still?

Also, are we putting users who are in the available window to ABTest or all users? If we put all, the number of users in A or B will be much higher than other.